src/slack.ts

```diff
@@ -888,6 +888,25 @@ export class App {
 		return reply;
 	}
 
+	public async listGroups(puppetId: number): Promise<IRetList[]> {
+		const p = this.puppets[puppetId];
+		if (!p) {
+			return [];
+		}
+		const reply: IRetList[] = [];
+		for (const [, team] of p.client.teams) {
+			if (team.partial) {
+				await team.load();
+			}
+			reply.push({
+				category: true,
```

Comment:
why is `category: true` needed here?

Reply:
It's probably not, but I wasn't sure what it should return exactly. The list rooms hook includes `category: true` for the teams it returns

Reply:
Yeah, the rooms list uses that for the heading so that it can list rooms in categories. So the category will just display that text and not linkify it or anything and thus not care about if it has an id or not

Reply:
should I remove the category here?